server/src/main/java/org/opensearch/action/admin/indices/stats/CommonStatsFlags.java

```diff
@@ -151,6 +167,14 @@ public Flag[] getFlags() {
         return flags.toArray(new Flag[0]);
     }
 
+    public EnumSet<CacheType> getIncludeCaches() {
```

Comment:
This should just return `Set<CacheType>`. The caller shouldn't need to know which set implementation is used.